pom.xml

```diff
@@ -47,56 +47,56 @@
 	</modules>
 
 	<properties>
-		<openMRSVersion>2.0.6</openMRSVersion>
+		<openMRSVersion>2.2.0</openMRSVersion>
 		<openmrsTestutilsVersion>1.3</openmrsTestutilsVersion>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<webservices.restModuleVersion>2.24.0</webservices.restModuleVersion>
+		<webservices.restModuleVersion>2.26.0</webservices.restModuleVersion>
 
 		<!-- Modules Bundled for Uganda distribution  -->
 		<!-- TODO: Arrange the dependencies and variables in alphabetical order -->
 		<!-- TODO: Remove dependencies on API where omods are available -->
-		<adminuiVersion>1.2.6-SNAPSHOT</adminuiVersion>
+		<adminuiVersion>1.3.0</adminuiVersion>
 		<addresshierarchyVersion>2.11.0</addresshierarchyVersion>
 		<allergyuiVersion>1.8.2-SNAPSHOT</allergyuiVersion>
-		<appframeworkVersion>2.11.0</appframeworkVersion>
+		<appframeworkVersion>2.13.0</appframeworkVersion>
 		<appuiVersion>1.11.0-SNAPSHOT</appuiVersion>
 		<appointmentschedulingVersion>1.7.0</appointmentschedulingVersion>
 		<appointmentschedulinguiVersion>1.9.0-SNAPSHOT</appointmentschedulinguiVersion>
 		<calculationVersion>1.2</calculationVersion>
-		<coreappsVersion>1.24.0-SNAPSHOT</coreappsVersion>
+		<coreappsVersion>1.24.0</coreappsVersion>
 		<databaseBackupVersion>1.3.0</databaseBackupVersion>
 		<dataEntryStatisticsVersion>1.6.0</dataEntryStatisticsVersion>
-		<dataexchangeVersion>1.3.2</dataexchangeVersion>
+		<dataexchangeVersion>1.3.3</dataexchangeVersion>
 		<dataintegrityVersion>4.4.1</dataintegrityVersion>
-		<emrapiVersion>1.27.0</emrapiVersion>
-		<eventVersion>2.6.0</eventVersion>
+		<emrapiVersion>1.28.0-SNAPSHOT</emrapiVersion>
+		<eventVersion>2.7.0</eventVersion>
 		<fhirVersion>1.13.0</fhirVersion>
 		<formentryappVersion>1.4.2</formentryappVersion>
 		<formfilterVersion>1.0.0</formfilterVersion>
-		<htmlformentryuiVersion>1.7.0</htmlformentryuiVersion>
-		<htmlformentryVersion>3.8.0</htmlformentryVersion>
-		<htmlwidgetsVersion>1.9.0</htmlwidgetsVersion>
+		<htmlformentryuiVersion>1.10.0-SNAPSHOT</htmlformentryuiVersion>
+		<htmlformentryVersion>3.9.2</htmlformentryVersion>
+		<htmlwidgetsVersion>1.10.0</htmlwidgetsVersion>
 		<idgenVersion>4.5.0</idgenVersion>
 		<jacksonVersion>1.8.3</jacksonVersion>
 		<junitParamsVersion>1.0.4</junitParamsVersion>
-		<legacyuiVersion>1.4.0</legacyuiVersion>
-		<metadatadeployVersion>1.10.0</metadatadeployVersion>
+		<legacyuiVersion>1.6.0</legacyuiVersion>
+		<metadatadeployVersion>1.11.0</metadatadeployVersion>
 		<metadatamappingVersion>1.3.4</metadatamappingVersion>
 		<metadatasharingVersion>1.5.0</metadatasharingVersion>
 		<openconceptlabVersion>1.1</openconceptlabVersion>
-		<openwebappsVersion>1.9.0</openwebappsVersion>
-		<patientFlagsVersion>3.0.1</patientFlagsVersion>
+		<openwebappsVersion>1.10.0</openwebappsVersion>
+		<patientFlagsVersion>3.0.2-SNAPSHOT</patientFlagsVersion>
```

Comment:
@ssmusoke change this to the released version of patient flags

Reply:
@slubwama done